packages/cms-cli/commands/create.js

```diff
@@ -128,12 +129,126 @@ const createTemplate = (name, dest, type = 'page-template') => {
   fs.copySync(assetPath, filePath);
 };
 
-function configureCreateCommand(program) {
+// Yargs Configuration
+const command = `${COMMAND_NAME} <type> <name> [dest]`;
+const describe = DESCRIPTION;
+const builder = yargs => {
+  addLoggerOptions(yargs, true);
+
+  yargs.positional('type', {
+    describe: 'Type of asset',
+    type: 'string',
+    choices: Object.values(TYPES),
+    demand: true,
+  });
+  yargs.positional('name', {
+    describe: 'Name of new asset',
+    type: 'string',
+  });
+  yargs.positional('dest', {
+    describe:
+      'Destination folder for the new asset, relative to your current working directory. If omitted, this argument will default to your current working directory.',
+    type: 'string',
+  });
+
+  return yargs;
+};
+const handler = async argv =>
+  action({ type: argv.type, name: argv.name, dest: argv.dest }, argv);
+
+const action = async ({ type, name, dest }, options) => {
+  setLogLevel(options);
+  logDebugInfo(options);
+  type = typeof type === 'string' && type.toLowerCase();
+
+  if (type === 'global-partial') {
+    logger.error(
+      `The asset type ${type} has been deprecated. Please choose the "template" asset and select "global partial".`
+    );
+    return;
+  }
+
+  if (!type || !TYPES[type]) {
+    logger.error(
+      `The asset type ${type} is not supported. Supported asset types are ${SUPPORTED_ASSET_TYPES}.`
+    );
+    return;
+  }
+
+  // TODO: In yargs use `.implies()`
```

Comment:
Should we do this?

Reply:
Based on what we discussed in our 1:1 I think we should hold off on this for now and rework these commands to be subcommands based on asset type.  I'll remove the comment for now